mithril-common/src/digesters/immutable_file.rs

```diff
@@ -218,7 +225,48 @@ mod tests {
     }
 
     #[test]
-    fn list_immutable_file_should_skip_last_number() {
+    fn list_all_immutable_file_should_not_skip_last_number() {
+        let target_dir =
+            get_test_dir("list_all_immutable_file_should_not_skip_last_number/immutable");
+        let entries = vec![
+            "123.chunk",
+            "123.primary",
+            "123.secondary",
+            "125.chunk",
+            "125.primary",
+            "125.secondary",
+            "0124.chunk",
+            "0124.primary",
+            "0124.secondary",
+            "223.chunk",
+            "223.primary",
+            "223.secondary",
+            "0423.chunk",
+            "0423.primary",
+            "0423.secondary",
+            "0424.chunk",
+            "0424.primary",
+            "0424.secondary",
+            "21.chunk",
+            "21.primary",
+            "21.secondary",
+        ];
+        create_fake_files(&target_dir, &entries);
+        let result = ImmutableFile::list_all_in_dir(target_dir.parent().unwrap())
+            .expect("ImmutableFile::list_in_dir Failed");
+
+        assert_eq!(result.last().unwrap().number, 424);
+        assert_eq!(
+            result.len(),
```

Comment:
We can create variables and use them directly in the message.
The `assert_eq` will be more readable and we are sure to use the same value in the assert and in the message (in particular in the next test in which we make some operation : list_completed_immutable_file_should_skip_last_number)

Reply:
Done